### PR TITLE
IGVF-243-sample-term-required-biosample

### PR DESCRIPTION
**Pull request by khzlin**

### CODE REVIEW

File: src/igvfd/tests/fixtures/schemas/primary_cell.py [hunk 1]
```diff
@@ -87,3 +90,12 @@ def primary_cell_v4(primary_cell, phenotype_term_alzheimers):
         'disease_term': phenotype_term_alzheimers['@id']
     })
     return item
+
+
+@pytest.fixture
+def primary_cell_v5(primary_cell):
+    item = primary_cell.copy()
+    item.update({
+        'schema_version': '6'
```
Comment (jenjou) [inline src/igvfd/tests/fixtures/schemas/primary_cell.py:99]: I think the schema version should be 5 here.

File: src/igvfd/tests/fixtures/schemas/differentiated_cell.py [hunk 2]
```diff
@@ -64,3 +65,12 @@ def differentiated_cell_v4(differentiated_cell, phenotype_term_alzheimers):
         'disease_term': phenotype_term_alzheimers['@id']
     })
     return item
+
+
+@pytest.fixture
+def differentiated_cell_v5(differentiated_cell):
+    item = differentiated_cell.copy()
+    item.update({
+        'schema_version': '6'
```
Comment (jenjou) [inline src/igvfd/tests/fixtures/schemas/differentiated_cell.py:74]: I think the schema version should be 5 here.

File: src/igvfd/schemas/changelogs/biosample.md [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
 
+### Schema version 5
+
+* Add `biosample_term` to requirements.
+
 ### Schema version 4
 
 * Rename `disease_term` to `disease_terms`.
 * Allow `disease_terms` to be an array of `phenotype_terms`.
-
```
Comment (jenjou) [inline src/igvfd/schemas/changelogs/biosample.md:7]: These blank lines can be kept as is (no need to remove)

File: src/igvfd/schemas/changelogs/whole_organism.md [hunk 4]
```diff
@@ -1,5 +1,8 @@
 ## Changelog for *`whole_organism.json`*
 
+### Schema version 5
+
+* Add `biosample_term` to requirements.
```
Comment (jenjou) [inline src/igvfd/schemas/changelogs/whole_organism.md:5]: We can add a blank line after this line

**Review by jenjou — CHANGES_REQUESTED** [review]
There's typos in 2 of the fixtures and some blank lines in the changelogs, otherwise looks ok.

**Review by jenjou — APPROVED** [review]

File: src/igvfd/schemas/changelogs/primary_cell.md [hunk 5]
```diff
@@ -1,6 +1,11 @@
 ## Changelog for *`primary_cell.json'*
 
+### Schema version 7
```
Comment (KAndreeva) [inline src/igvfd/schemas/changelogs/primary_cell.md:3]: The primary_cell.json indicates version 6, but this changelog indicates version 7.

File: src/igvfd/schemas/changelogs/biosample.md [hunk 6]
```diff
@@ -7,6 +7,7 @@
 * Add `upper_bound_age`.
 * Add `embryonic`.
 * Convert `age` to be calculated from `lower_bound_age` and `upper_bound_age` and not submittable.
+* Add `biosample_term` to requirements.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/biosample.md:10]: I think better would be "Require 'biosample_term'."

File: src/igvfd/schemas/changelogs/cell_line.md [hunk 7]
```diff
@@ -1,11 +1,13 @@
 ## Changelog for *`cell_line.json`*
 
 ### Schema version 6
+
 * Remove `life_stage`.
 * Add `lower_bound_age`.
 * Add `upper_bound_age`.
 * Add `embryonic`.
 * Convert `age` to be calculated from `lower_bound_age` and `upper_bound_age` and not submittable.
+* Add `biosample_term` to requirements.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/cell_line.md:10]: I think better would be "Require 'biosample_term'."

File: src/igvfd/schemas/changelogs/differentiated_cell.md [hunk 8]
```diff
@@ -1,11 +1,13 @@
 ## Changelog for *`differentiated_cell.json`*
 
 ### Schema version 6
+
 * Remove `life_stage`.
 * Add `lower_bound_age`.
 * Add `upper_bound_age`.
 * Add `embryonic`.
 * Convert `age` to be calculated from `lower_bound_age` and `upper_bound_age` and not submittable.
+* Add `biosample_term` to requirements.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/differentiated_cell.md:10]: I think better would be "Require 'biosample_term'."

File: src/igvfd/schemas/changelogs/differentiated_tissue.md [hunk 9]
```diff
@@ -1,11 +1,13 @@
 ## Changelog for *`differentiated_tissue.json`*
 
 ### Schema version 6
+
 * Remove `life_stage`.
 * Add `lower_bound_age`.
 * Add `upper_bound_age`.
 * Add `embryonic`.
 * Convert `age` to be calculated from `lower_bound_age` and `upper_bound_age` and not submittable.
+* Add `biosample_term` to requirements.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/differentiated_tissue.md:10]: I think better would be "Require 'biosample_term'."

File: src/igvfd/schemas/changelogs/primary_cell.md (hunk 5 above)
Comment (gabdank) [inline src/igvfd/schemas/changelogs/primary_cell.md:3]: Not sure the version should be 7, should not it be 6?
I think better would be "Require 'biosample_term'."

File: src/igvfd/schemas/changelogs/tissue.md [hunk 10]
```diff
@@ -1,11 +1,13 @@
 ## Changelog for *`tissue.json`*
 
 ### Schema version 6
+
 * Remove `life_stage`.
 * Add `lower_bound_age`.
 * Add `upper_bound_age`.
 * Add `embryonic`.
 * Convert `age` to be calculated from `lower_bound_age` and `upper_bound_age` and not submittable.
+* Add `biosample_term` to requirements.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/tissue.md:10]: I think better would be "Require 'biosample_term'."

File: src/igvfd/schemas/changelogs/whole_organism.md [hunk 11]
```diff
@@ -1,11 +1,13 @@
 ## Changelog for *`whole_organism.json`*
 
 ### Schema version 5
+
 * Remove `life_stage`.
 * Add `lower_bound_age`.
 * Add `upper_bound_age`.
 * Add `embryonic`.
 * Convert `age` to be calculated from `lower_bound_age` and `upper_bound_age` and not submittable.
+* Add `biosample_term` to requirements.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/whole_organism.md:10]: I think better would be "Require 'biosample_term'."

**Review by gabdank — CHANGES_REQUESTED** [review]
small comments on changelogs

**Review by gabdank — CHANGES_REQUESTED** [review]
cell_line.json says version 6, but cell_line changelog says 7. it should say 6.

**Review by gabdank — APPROVED** [review]

**Pull request merged**